testsuites/syncgateway/functional/tests/conftest.py

```diff
@@ -480,6 +483,11 @@ def params_from_base_suite_setup(request):
         expected_sync_gateway_version=sync_gateway_version
     )
 
+    need_sgw_admin_auth = (not disable_admin_auth) and sync_gateway_version >= "3.0"
+    print("==============================================")
```

Comment:
Clean up prints

Reply:
cleaned up the print statement, and added proper log_info